terraform/plugin_cache.tf

```diff
@@ -0,0 +1,12 @@
+provider "aws" {
+  version = "~> 1.6"
```

Comment:
does it make sense to just use [exact version equality](https://www.terraform.io/docs/configuration/terraform.html#specifying-a-required-terraform-version)?

Reply:
Sure. I've pinned this to the versions that the DockerHub build chose https://hub.docker.com/r/governmentpaas/terraform/builds/bxwtfu2lbnwwqdjt8ktmynw/ :-)